docs/src/description/buses.md

```diff
@@ -0,0 +1,63 @@
+# Buses
+
+A bus is a construct which aims to simplify description of non-local constraints defined via multiset or LogUp checks.
+
+## Bus types
+
+- Multiset (`unit`): Multiset-based buses can represent constraints which specify values that must have been added or removed from a column, in no particular order.
+[Incremental Multiset Hash Functions and Their Application to Memory Integrity Checking - Clarke et al. MIT CSAIL (2018)](https://people.csail.mit.edu/devadas/pubs/mhashes.pdf)
```

Comment:
I would also add a link to [our docs](https://0xpolygonmiden.github.io/miden-vm/design/lookups/multiset.html) somewhere here.